packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -51,10 +51,6 @@ class BuildAotCommand extends BuildSubCommand with TargetPlatformBasedDevelopmen
         help: 'Build the AOT bundle with bitcode. Requires a compatible bitcode engine.',
         hide: true,
       );
-    // --track-widget-creation is exposed as a flag here to deal with build
```

Comment:
If it's not supported in this mode, why is it included in build invalidation logic?

Reply:
what?

Reply:
This comment block confuses me.  I might be misreading it.  It seems to say:

```
Allow this flag that has no effect, other than to invalidate the build.
```

Reply:
In the past when the build aot command would use, sometimes gradle/xcode would pass track-widget-creation=false which would crash the tool since the argument did not exist

Reply:
So instead should we be asserting that it is either not passed or passed as false?

Reply:
No, because we don't use this command today. This is just to restore some of the Dart teams benchmarks until I get them to switch to build apk/ios so I can remove it

Reply:
So what happens with this patch if I, from the command line, run `flutter build aot --track-widget-creation`?

Reply:
Something something there is no option called "--track-widget-creation"